ecs-image-build/Dockerfile

```diff
@@ -0,0 +1,12 @@
+ARG IMAGE_VERSION="latest"
+FROM 416670754337.dkr.ecr.eu-west-2.amazonaws.com/ci-open-jdk8-maven-build:${IMAGE_VERSION}
+
+ARG APPLICATION_NAME=docs.developer.ch.gov.uk
+ARG PORT="8080"
+ARG JAVA_MEM_ARGS=""
```

Comment:
These build arguments aren't referenced further in the Dockerfile - are they required / should they be assigned to `ENV`?